gradle/verification-metadata.xml

```diff
@@ -111,6 +111,14 @@
             <sha256 value="c80313e6c26c0b9e0c241504718e2d8bbc2798b73429933adf03fdc6d84f0e70" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="junit" name="junit" version="4.13.2">
+         <artifact name="junit-4.13.2.jar">
+            <sha256 value="8e495b634469d64fb8acfa3495a065cbacc8a0fff55ce1e31007be4c16dc57d3" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="junit-4.13.2.pom">
+            <sha256 value="569b6977ee4603c965c1c46c3058fa6e969291b0160eb6964dd092cd89eadd94" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
```

Comment:
Why the dependency on junit 4.13.2?

Reply:
The vintage engine pulls this in, so I think we may be forced to upgrade from 4.8.2 JUnit 4 to 4.13.2 JUnit 4 as well, although the current setup seems to work (I remember some Rules being deprecated in later JUnit 4, specifically rules for catching exceptions which could require rewriting those tests to use `assertThrows`).

Reply:
Assuming this is okay now as approvals have been received.

Reply:
Yes, this is fine.